build/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM registry.access.redhat.com/ubi8/ubi
-RUN curl https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo > /etc/yum.repos.d/bazel.repo && \
```

Comment:
In fact I assume that the bazel installation could be removed or commented here.
It was mainly when I wanted to experiment the on-the-fly creation of docker images with all the plugin artifacts injected inside.
But at this time I didn't have time to finish this experiment, and it's not currently prioritized. 
So this bazel-related code should probably be moved to a branch.

Reply:
bazel installation is removed now.